source/base/TimeSpan.ooc

```diff
@@ -21,6 +21,7 @@ TimeSpan: cover {
 	toHours: func -> Long { this ticks / DateTime ticksPerHour }
 	toDays: func -> Long { this ticks / DateTime ticksPerDay }
 	toWeeks: func -> Long { this ticks / DateTime ticksPerWeek }
+	inRange: func (start, end: This) -> Bool { this >= start && this <= end }
```

Comment:
We do have the similar `in` defined on numeric types (`6 in(2 .. 8) == true`) but using a keyword as method name isn't good practice, methinks. So if you're happy with `inRange` I won't bother you about it.